README.md

```diff
@@ -5,641 +5,291 @@
+- Include the necessary
+  maven [dependencies for Open AI Chat Completion](docs/guides/OPENAI_CHAT_COMPLETION.md#maven-dependencies) in
+  your `pom.xml`.
 
-```java
-final var llmConfig = LLMModuleConfig.create().modelName("gpt-35-turbo").modelParams(Map.of());
-
-final var inputParams =
-    Map.of("input", "Reply with 'Orchestration Service is working!' in German");
-final var template = ChatMessage.create().role("user").content("{{?input}}");
-final var templatingConfig = TemplatingModuleConfig.create().template(template);
-
-final var config =
-    CompletionPostRequest.create()
-        .orchestrationConfig(
-            OrchestrationConfig.create()
-                .moduleConfigurations(
-                    ModuleConfigs.create()
-                        .llmModuleConfig(llmConfig)
-                        .templatingModuleConfig(templatingConfig)))
-        .inputParams(inputParams);
-
-final CompletionPostResponse result =
-    new OrchestrationCompletionApi(getOrchestrationClient("default"))
-        .orchestrationV1EndpointsCreate(config);
-
-final String messageResult =
-    result.getOrchestrationResult().getChoices().get(0).getMessage().getContent();
-```
-
-See [an example in our Spring Boot application](sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java)
+### Write the Code
 
-### Messages history
+Create a Java class (e.g., `OpenAiExample.java`) and include the following code:
 
 ```java
-final var llmConfig = LLMModuleConfig.create().modelName("gpt-35-turbo").modelParams(Map.of());
-
-List<ChatMessage> messagesHistory =
-    List.of(
-        ChatMessage.create().role("user").content("What is the capital of France?"),
-        ChatMessage.create().role("assistant").content("The capital of France is Paris."));
-
-final var message =
-    ChatMessage.create().role("user").content("What is the typical food there?");
-final var templatingConfig = TemplatingModuleConfig.create().template(message);
-
-final var config =
-    CompletionPostRequest.create()
-        .orchestrationConfig(
-            OrchestrationConfig.create()
-                .moduleConfigurations(
-                    ModuleConfigs.create()
-                        .llmModuleConfig(llmConfig)
-                        .templatingModuleConfig(templatingConfig)))
-        .inputParams(Map.of())
-        .messagesHistory(messagesHistory);
-
-final CompletionPostResponse result =
-    new OrchestrationCompletionApi(getOrchestrationClient("default"))
-        .orchestrationV1EndpointsCreate(config);
-
-final String messageResult =
-    result.getOrchestrationResult().getChoices().get(0).getMessage().getContent();
+import com.sap.ai.sdk.foundationmodels.openai.OpenAiClient;
+import com.sap.ai.sdk.foundationmodels.openai.OpenAiModel;
+import com.sap.ai.sdk.foundationmodels.openai.model.OpenAiChatCompletionOutput;
+
+public class OpenAiExample {
+  public static void main(String[] args) {
```

Comment:
(Minor) 

Please remove imports and psvm (`public static void main`) boilerplate. The code is easy enough, I think.